src/libraries/Common/src/Interop/Windows/Gdi32/Interop.CreateDC.cs

```diff
@@ -5,11 +5,13 @@
 using System;
 using System.Runtime.InteropServices;
 
+#nullable enable
```

Comment:
I believe this file is only used in `System.Drawing.Common` so might not need to add `#nullable enable` here.

Reply:
Can't hurt to add to common dependencies though? We'll remove those eventually anyway

Reply:
Even it wouldn't hurt we would need to remove it from lots of files, for me better to have it less when possible